backend/tests/services/knowledge/test_artifact_service.py

```
@@ -0,0 +1,624 @@
+    result = ArtifactService._parse_content(
+        KnowledgeArtifactType.MIND_MAP,
+        content,
+    )
+
+    assert '"root_id":"root"' in result
+    assert '"parent_id":"root"' in result
+
+
+def test_parse_mind_map_accepts_explanatory_text_around_json():
+    content = """
+    已根据所选文档生成思维导图：
```

Ruff (0.15.21)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)